README.md

```diff
@@ -179,6 +193,10 @@ Q: Why does the script give an error about not being able to unload modules ftdi
 
 A: In order to flash the device, no other program or device on the system can be using these kernel modules or the device. Stop any program/container that could be using the modules or device (likely deCONZ) and then invoke the script again. If the error persists, you may need to temporarily remove other USB serial devices from the system in order allow the script to completely unload the kernel modules.
 
+Q: Why does a flush run fail after some seconds even if I specified a timeout much longer?
```

Comment:
😁 this typo was unintended - you don't have to `flush` your device if the `flash` runs into a timeout!